src/components/approval-tasks/approval-notification/ApprovalToastContent.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ExternalLink } from '../../utils/link';
+
+import './ApprovalToastContent.scss';
+
+interface ApprovalToastContentProps {
+  type: string;
+  uniquePipelineRuns: number;
+  devconsolePath?: string;
+  adminconsolePath?: string;
+}
+
+const ApprovalToastContent: React.FC<ApprovalToastContentProps> = ({
+  type,
+  uniquePipelineRuns,
+  devconsolePath,
+  adminconsolePath,
+}) => {
+  const { t } = useTranslation('plugin__pipelines-console-plugin');
+  if (type === 'current') {
+    return (
+      <>
+        {t('Your approval has been requested on {{plrs}} pipeline', {
+          plrs: uniquePipelineRuns,
+        })}
+        {uniquePipelineRuns > 1 ? t('runs.') : t('run.')}
```

Comment:
Please take a look how we can use the [i18next plurals](https://www.i18next.com/translation-function/plurals) feature here. I'm fine to handle this in a follow up PR. But we should improve this.